modules/sdk-core/src/bitgo/utils/tss/baseTSSUtils.ts

```diff
@@ -268,12 +248,10 @@ export default class BaseTssUtils<KeyShare> extends MpcUtils implements ITssUtil
       preview,
     };
 
-    const txRequest = (await this.bitgo
+    return (await this.bitgo
```

Comment:
You don't have to `await` here since `.result()` is already returning a promise

Reply:
I didn't make any changes here. You're suggestion makes sense, but i'd rather not change code not related to this PR.